static/js/components/channels/ChannelCreateDialog.js

```diff
@@ -0,0 +1,145 @@
+    }
+  }
+
+  renderChannelFilters(program: AvailableProgram, filters: ?Array<Filter>) {
+    if (!filters || R.isEmpty(filters)) {
+      return (
+        <div className="sk-selected-filters-list">
+          <div className="sk-selected-filters-option sk-selected-filters__item">
+            <div className="sk-selected-filters-option__name">
+              All users in {program.title}
+            </div>
+          </div>
+        </div>
+      )
+    }
+    return (
+      <div className="sk-selected-filters-list">
+        {renderFilterOptions(filters)}
+      </div>
+    )
+  }
+
+  render() {
+    const {
+      channelDialog: { inputs, filters, validationErrors, fetchStatus },
+      dialogVisibility,
+      currentProgramEnrollment,
+      updateEmailFieldEdit
+    } = this.props
+
+    if (!currentProgramEnrollment) {
+      return null
+    }
+
+    return (
+      <Dialog
+        title="Create New Channel"
+        titleClassName="dialog-title"
+        contentClassName="dialog create-channel-dialog"
+        className="create-channel-dialog-wrapper"
+        open={dialogVisibility}
+        onRequestClose={this.closeAndClear}
+        actions={dialogActions(
+          this.closeAndClear,
+          this.closeAndCreate,
+          fetchStatus === FETCH_PROCESSING,
+          "Create",
+          "",
+          !R.isEmpty(validationErrors)
+        )}
+      >
+        <Grid>
+          <Cell col={12}>
+            <p>This channel is for:</p>
+            {this.renderChannelFilters(currentProgramEnrollment, filters)}
+          </Cell>
+          <Cell col={12}>
+            <TextField
+              floatingLabelText="Title"
+              name="title"
+              value={inputs.title || ""}
+              fullWidth={true}
+              onChange={updateEmailFieldEdit("title")}
+              maxLength={100}
+            />
+            {this.showValidationError("title")}
+          </Cell>
+          <Cell col={12}>
+            <TextField
+              floatingLabelText="Name"
+              name="name"
+              value={inputs.name || ""}
+              onChange={updateEmailFieldEdit("name")}
+              fullWidth={true}
+              maxLength={21}
+            />
+            {this.showValidationError("name")}
+            <p>
+              No spaces, e.g., "books" or "bookclub". Avoid using solely
+              trademarked names, e.g. use "FansOfAcme" instead of "Acme". Once
+              chosen, this name cannot be changed.
+            </p>
+          </Cell>
+          <Cell col={12}>
+            <TextField
+              floatingLabelText="Description"
+              name="public_description"
+              value={inputs.public_description || ""}
+              onChange={updateEmailFieldEdit("public_description")}
+              fullWidth={true}
+              multiLine={true}
+              maxLength={500}
+            />
+            {this.showValidationError("public_description")}
```

Comment:
Is there any validation message which currently appears for this field?

Reply:
it's required